src/rxreact.tsx

```diff
@@ -30,7 +30,7 @@ function withViewModelFactory<S, A, P extends S & ActionMap<A>>(
       constructor(props: Difference<P, S & ActionMap<A>>) {
         super(props);
         let viewModel = viewModelFactory(this.propsObservable);
-        this.observableState = combineObservables(viewModel.inputs);
+        this.observableState = Object.keys(viewModel.inputs).length ? combineObservables(viewModel.inputs) : new Observable();
```

Comment:
You need an observable that emits something.

I recommend using the `of` constructor -- you'll need to add it to the rxjs import above

```suggestion
        this.observableState = Object.keys(viewModel.inputs).length ? combineObservables(viewModel.inputs) : of({});
```